python/py-oci/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-oci
+version             2.113.0
+revision            0
+
+categories-append   net
+
+license             UPL-1 Apache-2
+platforms           {darwin any}
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         Oracle Cloud Infrastructure Python SDK
+long_description    {*}${description}
+
+homepage            https://docs.oracle.com/en-us/iaas/tools/python/latest/index.html
+
+checksums           md5     fd303b61da35053c6f249fbe82b53a7d \
+                    rmd160  535d9c52c75e811973421484c604dc25db15e153 \
+                    sha256  dc5b5e6410d729fde41d67912fbf9177af4efe354c10d558076af27643e349d5 \
+                    size    11396545
+
+python.versions     38 39
```

Our currently supported version of Python is Python 3.11.  Is it possible to extend this port up to that version?

```suggestion
python.versions     38 39 310 311
```

---

Although it _seems to_ work, Oracle officially supports 3.8 and 3.9 only (see https://github.com/oracle/oci-python-sdk#about)